windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,23 +17,46 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
-[id="wmco-1-0-2"]
-== Release notes for Red Hat Windows Machine Config Operator 1.0.2
+[id="wmco-2-0-0"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.0
 
-This release of the WMCO provides initial support for running Windows compute nodes in an {product-title} cluster.
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 2.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:0440[RHBA-2021:0440].
 
 WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
 
 * Amazon Web Services (AWS)
 * Microsoft Azure
+* VMware vSphere
 
-The following Windows Server operating systems are supported in the initial release of the WMCO:
+The following Windows Server operating systems are supported in this release of the WMCO:
 
 * Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
 
-[id="wmco-1-0-1-known-issues"]
+The WMCO 2.x version is only compatible with {product-title} 4.7.
+
+[id="wmco-2-0-0-enhancements"]
+=== New features and enhancements
+
+This release adds the following new features and improvements.
+
+[id="wmco-2-0-0-vsphere-support"]
+==== Support for clusters running on VMware vSphere
+
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create Windows compute nodes by creating a Windows `MachineSet` object on vSphere to host Windows Server compute machines. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
```

Comment:
I was wondering if there is a difference between "compute nodes" and "compute machines" in this sentence. If there is not, you could use only one of these references for consistency in this paragraph. If they are the same thing, you could minimise the sentence as follows:

"You can create a Windows `MachineSet` object on vSphere to host Windows Server compute nodes."